src/backend/joanie/tests/core/test_command_delete_stuck_orders.py

```diff
@@ -0,0 +1,103 @@
+"""Tests for the `delete_stuck_orders` management command."""
+
+from datetime import timedelta
+from unittest import mock
+
+from django.conf import settings
+from django.core.management import call_command
+
+from joanie.core import enums, factories, models
+from joanie.tests.base import LoggingTestCase
+
+
+class DeleteStuckOrdersCommandTestCase(LoggingTestCase):
+    """Test case for the management command `delete_stuck_orders`."""
+
+    def test_commands_delete_stuck_orders_signing_states(self):  # pylint: disable=too-many-locals
```

Comment:
I believe this test should only assert that the function is called and that the returned variables are used in logs.